gameserver/GameBoard/Cards/CardScore.ts

```diff
@@ -0,0 +1,29 @@
+import {CARD_VALUE} from "./CardValue";
+import {CARDS} from "./Cards";
+import {Deck} from "./Deck"
+import {CARD_COLOR} from "./CardColor";
+
+let basicScore = [
+    [CARD_VALUE.SIX, 0],
+    [CARD_VALUE.SEVEN, 0],
+    [CARD_VALUE.HEIGHT, 0],
+    [CARD_VALUE.NINE, 0],
+    [CARD_VALUE.TEN, 10],
+    [CARD_VALUE.JACK, 2],
+    [CARD_VALUE.QUEEN, 3],
+    [CARD_VALUE.KING, 4],
+    [CARD_VALUE.ACE, 11]
+];
```

Comment:
Looks strange, maybe an object would be better ?

Maybe something like this
```js
let basicScore ⁼ {}
basicScore[CARD_VALUE.SIX] = 0;
basicScore[CARD_VALUE.SEVEN] = 0;
basicScore[CARD_VALUE.EIGHT] = 0;
basicScore[CARD_VALUE.NINE] = 0;
basicScore[CARD_VALUE.TEN] = 10;
basicScore[CARD_VALUE.JACK] = 2;
basicScore[CARD_VALUE.QUEEN] = 3;
basicScore[CARD_VALUE.KING] = 4;
basicScore[CARD_VALUE.ACE] = 11;
```

@BlueSpaceCookie your opinion on this ?

Reply:
I prefer the class version, better lisibility